error on `make create-package-deb`

When attempting to create a debian package from the latest master (e8081bb), I am receiving the error `No rule to make target 'pre-create-package', needed by 'create-package-deb'. `.

A regular `make` is working for me.  Logs below.

```
seno@senobox:~/xdotool$ make clean
rm -f *.o xdotool xdotool.static xdotool.1 xdotool.html \
      libxdo.so libxdo.so.3 libxdo.a libxdo.pc

seno@senobox:~/xdotool$ make
pod2man -c "" -r "" xdotool.pod > xdotool.1
cc -pipe -O2 -pedantic -Wall -W -Wundef -Wendif-labels -Wshadow -Wpointer-arith -Wbad-function-cast -Wcast-align -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wdisabled-optimization -Wno-missing-field-initializers -g    -std=c99  -fPIC -c xdo.c
xdo.c:2042:12: warning: ‘appears_to_be_wayland’ defined but not used [-Wunused-function]
 static int appears_to_be_wayland(Display *xdpy) {
            ^~~~~~~~~~~~~~~~~~~~~
cc -pipe -O2 -pedantic -Wall -W -Wundef -Wendif-labels -Wshadow -Wpointer-arith -Wbad-function-cast -Wcast-align -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wdisabled-optimization -Wno-missing-field-initializers -g    -std=c99  -fPIC -c xdo_search.c
....
cc -pipe -O2 -pedantic -Wall -W -Wundef -Wendif-labels -Wshadow -Wpointer-arith -Wbad-function-cast -Wcast-align -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wdisabled-optimization -Wno-missing-field-initializers -g    -std=c99    -c -o cmd_get_display_geometry.o cmd_get_display_geometry.c
cc -o xdotool xdotool.o cmd_click.o cmd_mousemove.o cmd_mousemove_relative.o cmd_mousedown.o cmd_mouseup.o cmd_getmouselocation.o cmd_type.o cmd_key.o cmd_windowmove.o cmd_windowactivate.o cmd_windowfocus.o cmd_windowraise.o cmd_windowsize.o cmd_windowstate.o cmd_set_window.o cmd_search.o cmd_getwindowfocus.o cmd_getwindowpid.o cmd_getactivewindow.o cmd_windowmap.o cmd_windowunmap.o cmd_windowreparent.o cmd_set_num_desktops.o cmd_get_num_desktops.o cmd_set_desktop.o cmd_get_desktop.o cmd_set_desktop_for_window.o cmd_get_desktop_for_window.o cmd_get_desktop_viewport.o cmd_set_desktop_viewport.o cmd_windowkill.o cmd_behave.o cmd_window_select.o cmd_getwindowname.o cmd_getwindowclassname.o cmd_behave_screen_edge.o cmd_windowminimize.o cmd_exec.o cmd_getwindowgeometry.o cmd_windowclose.o cmd_windowquit.o cmd_sleep.o cmd_get_display_geometry.o -L. -lxdo -Xlinker -rpath /usr/local/lib  -lm -lX11  -lrt
```

However, when I then try to create the debian package, I receive:

```
seno@senobox:~/xdotool$ make create-package-deb
make: *** No rule to make target 'pre-create-package', needed by 'create-package-deb'.  Stop.
```

If I modify the `Makefile` to replace the `pre-create-package` target, it gets further, but still errors out.  Logs:

```
root@senobox:/home/seno/xdotool# git diff
diff --git a/Makefile b/Makefile
index c28dfcc..5424f4c 100644
--- a/Makefile
+++ b/Makefile
@@ -235,7 +235,7 @@ test-package-build: create-package
 # tarballs.

 DEBDIR=deb-build
-create-package-deb: pre-create-package VERSION xdo_version.h
+create-package-deb: test-package-build VERSION xdo_version.h
        [ -d $(DEBDIR) ] && rm -r $(DEBDIR) || true
        $(MAKE) xdotool.deb xdotool-doc.deb libxdo$(MAJOR).deb libxdo$(MAJOR)-dev.deb

root@senobox:/home/seno/xdotool# make create-package-deb
echo "Creating package: xdotool-3.20210903.1"
Creating package: xdotool-3.20210903.1
mkdir "xdotool-3.20210903.1"
rsync --exclude '.*' -a `ls -d *.pod COPYRIGHT *.c *.h *.pc examples t CHANGELIST README.md Makefile* version.sh platform.sh cflags.sh VERSION Doxyfile 2> /dev/null` "xdotool-3.20210903.1/"
tar -zcf "xdotool-3.20210903.1.tar.gz" "xdotool-3.20210903.1"
rm -r "xdotool-3.20210903.1"
echo "Testing package xdotool-3.20210903.1"
Testing package xdotool-3.20210903.1
tar -zxf xdotool-3.20210903.1.tar.gz
make -C ./xdotool-3.20210903.1
make[1]: Entering directory '/home/seno/xdotool/xdotool-3.20210903.1'
pod2man -c "" -r "" xdotool.pod > xdotool.1
cc -pipe -O2 -pedantic -Wall -W -Wundef -Wendif-labels -Wshadow -Wpointer-arith -Wbad-function-cast -Wcast-align -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wdisabled-optimization -
....
make[1]: Leaving directory '/home/seno/xdotool/xdotool-3.20210903.1'
rm -rf ./xdotool-3.20210903.1
echo "Package ready: xdotool-3.20210903.1";
Package ready: xdotool-3.20210903.1
[ -d deb-build ] && rm -r deb-build || true
make xdotool.deb xdotool-doc.deb libxdo3.deb libxdo3-dev.deb
make[1]: Entering directory '/home/seno/xdotool'
make install DESTDIR=deb-build PREFIX=/usr INSTALLMAN=/usr/share/man
make[2]: Entering directory '/home/seno/xdotool'
install -d deb-build/usr
install -d deb-build/usr/lib
install libxdo.so deb-build/usr/lib/libxdo.so.3
ln -sf libxdo.so.3 deb-build/usr/lib/libxdo.so
install -d deb-build/usr/bin
install -m 755 xdotool deb-build/usr/bin/
install -d deb-build/usr/share/man/man1
install -m 644 xdotool.1 deb-build/usr/share/man/man1/
install -d deb-build/usr/include
install -m 0644 xdo.h deb-build/usr/include/xdo.h
install -d deb-build/usr/lib/pkgconfig
install -m 0644 libxdo.pc deb-build/usr/lib/pkgconfig/libxdo.pc
Running ldconfig to update library cache
ldconfig: /usr/lib/x86_64-linux-gnu/libxdo.so.3 is not a symbolic link

make[2]: Leaving directory '/home/seno/xdotool'
make deb-build/xdotool/data.tar.gz deb-build/xdotool/control.tar.gz \
        deb-build/xdotool/debian-binary
make[2]: Entering directory '/home/seno/xdotool'
make[2]: *** No rule to make target 'deb-build/xdotool/', needed by 'deb-build/xdotool/data.tar.gz'.  Stop.
make[2]: Leaving directory '/home/seno/xdotool'
Makefile:243: recipe for target 'xdotool.deb' failed
make[1]: *** [xdotool.deb] Error 2
make[1]: Leaving directory '/home/seno/xdotool'
Makefile:239: recipe for target 'create-package-deb' failed
make: *** [create-package-deb] Error 2
```

Running Debian 9.8.

--- a/.gitignore
+++ b/.gitignore
@@ -3,3 +3,5 @@
 *.1
 libxdo.pc
 xdotool
+deb-build
+*.deb

--- a/Makefile
+++ b/Makefile
@@ -124,7 +124,8 @@ uninstall:
 .PHONY: clean
 clean:
 	rm -f *.o xdotool xdotool.static xdotool.1 xdotool.html \
-	      libxdo.$(LIBSUFFIX) libxdo.$(VERLIBSUFFIX) libxdo.a libxdo.pc
+	      libxdo.$(LIBSUFFIX) libxdo.$(VERLIBSUFFIX) libxdo.a libxdo.pc \
+	      *.deb
 
 xdo.o: xdo.c xdo_version.h
 	$(CC) $(CFLAGS) -fPIC -c xdo.c

--- a/Makefile
+++ b/Makefile
@@ -235,7 +235,7 @@ test-package-build: create-package
 # tarballs.
 
 DEBDIR=deb-build
-create-package-deb: pre-create-package VERSION xdo_version.h
+create-package-deb: VERSION xdo_version.h
 	[ -d $(DEBDIR) ] && rm -r $(DEBDIR) || true
 	$(MAKE) xdotool.deb xdotool-doc.deb libxdo$(MAJOR).deb libxdo$(MAJOR)-dev.deb
 
@@ -270,18 +270,18 @@ $(DEBDIR)/%/control.tar.gz: $(DEBDIR)/%/control $(DEBDIR)/%/md5sums
 	tar -C $(DEBDIR)/$* -zcf $(DEBDIR)/$*/control.tar.gz control md5sums 
 
 # Build a tarball for xdotool files
-$(DEBDIR)/xdotool/data.tar.gz: $(DEBDIR)/xdotool/
+$(DEBDIR)/xdotool/data.tar.gz: $(DEBDIR)/xdotool
 	tar -C $(DEBDIR) -zcf $@ usr/bin
 
 # Build a tarball for libxdo# files
-$(DEBDIR)/libxdo$(MAJOR)/data.tar.gz: $(DEBDIR)/libxdo$(MAJOR)/
+$(DEBDIR)/libxdo$(MAJOR)/data.tar.gz: $(DEBDIR)/libxdo$(MAJOR)
 	tar -C $(DEBDIR) -zcf $@ usr/lib
 
 # Build a tarball for libxdo#-dev files
-$(DEBDIR)/libxdo$(MAJOR)-dev/data.tar.gz: $(DEBDIR)/libxdo$(MAJOR)-dev/
+$(DEBDIR)/libxdo$(MAJOR)-dev/data.tar.gz: $(DEBDIR)/libxdo$(MAJOR)-dev
 	tar -C $(DEBDIR) -zcf $@ usr/include
 
 # Build a tarball for xdotool-doc files
-$(DEBDIR)/xdotool-doc/data.tar.gz: $(DEBDIR)/xdotool-doc/
+$(DEBDIR)/xdotool-doc/data.tar.gz: $(DEBDIR)/xdotool-doc
 	tar -C $(DEBDIR) -zcf $@ usr/share
 

